common/scanner.h

```diff
@@ -142,6 +143,21 @@ static inline bool external_scanner_scan(void *payload, TSLexer *lexer, const bo
     }
 
     return true;
+  } else if (valid_symbols[FUNCTION_SIGNATURE_SEMICOLON]) {
+    while (lexer->lookahead == '\n') {
+      advance(lexer);
```

Comment:
I think this could be `skip(lexer)`. From the docs

```
void (*advance)(TSLexer *, bool skip) - A function for advancing to the next character. If you pass true for the second argument, the current character will be treated as whitespace.
```